src/components/views/messages/MVideoBody.tsx

```diff
@@ -240,6 +240,8 @@ export default class MVideoBody extends React.PureComponent<IBodyProps, IState>
         let width = null;
         let poster = null;
         let preload = "metadata";
+        let loop = false;
+        let controls = true;
```

Comment:
```suggestion
        const loop = content.info["fi.mau.loop"];
        const controls = !content.info["fi.mau.hide_controls"];
```

Or similar feels more readable. 

Also, could you please add some comment explaining why is this necessary?